### DISCUSSION TITLE
signoz log-management timeline

### DISCUSSION THREAD

**Discussion by gayatrigunnala** [SigNoz/signoz#1296] [Q&A] [answered]
Can somebody help me with update on timeline of log-management UI support in signoz.

**Comment by ankitnayan** [comment, marked as answer]
@gayatrigunnala we are working on PRD. Would be great to scope down the full log management feature set into priorities. A few of the features I can think of are:
- live tail of logs from different sources
- filtering the logs using some filters parsed from the logline
- UI to select and parse a value from logline and create an index for faster filtering
- aggregation capabilities on filtered logs and plotting a chart
- configure retention period of logs on disk

### Would need a few inputs from those waiting for log management solution
1. What are the top features you need to consider a fully functional and useful log management tool
2. Which sources of logs do you mostly ingest? Application/container/syslog/k8s and more. We need to prioritise to using only a few with the initial launch.


After PRD freeze it should take 3 weeks to release. Planning to freeze the PRD in a week. So expected release timeline for logs would be around end of 3rd week of July (around 21st July).

@pranay01 @nityanandagohain

**Comment by pranay01** [comment]
hey @gayatrigunnala 

Just posted early designs for logs [here](https://github.com/SigNoz/signoz/issues/1139#issuecomment-1171923399) - would love to know if you have any feedback
